04_diy_neural/README.md

```diff
@@ -34,7 +34,19 @@
 * Video: [Data visualization for machine learning](https://vimeo.com/304131671)
 
 ### Assignment 4A due Wednesday, Sept 24, 9am
-* TBA (find and link to a dataset)
+1. Find a dataset that interests you. Some ideas:
+    * Something you find online. For example, take a look at [Kaggle](https://www.kaggle.com/) and you can find a list of datasets we've compiled on the [dataset wiki](https://github.com/ml5js/Intro-ML-Arts-IMA/wiki/Datasets).
+    * Find aa dataset that you collect yourself or is already being collected about you. For example, personal data like steps taken per day, browser history, minutes spent on your mobile device, sensor readings, and more.
+    * A dataset that you collect by crowdsourcing data. One way to approach this is to create a google form with questions and ask friends and fellow students to complete the form.
+2. After you have found or collected your dataset, write a blog post that addresses these questions:
+    * What is (are!) the data?
+    * What format is the data? (CSV, JSON, PDF, or . . )
+    * What are the dimensions of the data (rows and columns) and what are the data types (numbers, strings, images, etc.)?
```

Comment:
Should we explicitly ask what the variables are? It sometimes helps me better understand what I'm working with when I have to go through each one. But maybe that's too much to add to this list at the moment!

Reply:
Yes, I agree! I didn't use the term "variables" b/c I find it a little confusing in the context of programming. But it's important here. How about the following?

* What are the dimensions of the data (rows and columns)?
* What are the "variables" (also known as "data items"). In a CSV these would be the column headings. Do you recognize the data types (numbers, strings, images, etc.)?